packages/framework-provider-aws-infrastructure/test/infrastructure/deploy.test.ts

```diff
@@ -118,7 +118,7 @@ describe('the deployment process', () => {
 
       // Just checks that the assemble method does not fail, meaning that the stack is build correctly according to the
       // AWS validations
-      expect(() => Infrastructure.assemble(config)).not.to.throw()
+      expect(() => Infrastructure.assemble(config, [])).not.to.throw()
```

Comment:
Perhaps its better to keep consistency in parameter order, and make `assemble` to accept the array as a first parameter, like `deploy`?

Reply:
I'll do the opposite because it makes sense that the array of plugins is optional.